client/src/firebase/functions/src/index.ts

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable no-throw-literal */
+/* eslint-disable linebreak-style */
+/* eslint-disable max-len */
+/**
+ * Import function triggers from their respective submodules:
+ *
+ * import {onCall} from "firebase-functions/v2/https";
+ * import {onDocumentWritten} from "firebase-functions/v2/firestore";
+ *
+ * See a full list of supported triggers at https://firebase.google.com/docs/functions
+ */
+
+import * as functions from "firebase-functions";
+import * as logger from "firebase-functions/logger";
+import * as admin from "firebase-admin";
+import * as serviceAccount from "./callforhelp-37002-firebase-adminsdk-7ivya-e3e03f01cb.json";
+
+const DATABASE_URL = "https://callforhelp-37002-default-rtdb.firebaseio.com";
```

Comment:
Is this here on purpose?

Reply:
What is this? If it's the DATABASE_URL then yes